packages/open-next/src/plugins/edge.ts

```diff
@@ -39,7 +39,9 @@ export function openNextEdgePlugins({
   isInCloudflare,
 }: IPluginSettings): Plugin {
   const entryFiles =
-    middlewareInfo?.files.map((file: string) => path.join(nextDir, file)) ?? [];
+    middlewareInfo?.files.map((file: string) =>
+      path.join(nextDir, file).replace(/\\/g, "/"),
```

Comment:
nit: Maybe we could introduce a `normalizePath` function to make it easier to grep locations where we normalize the paths?